src/uniprotkb/components/protein-data-views/FeaturesView.tsx

```diff
@@ -11,6 +11,7 @@ import FeatureType from '../../types/featureType';
 import { UniProtProtvistaEvidenceTag } from './UniProtKBEvidenceTag';
 import FeaturesTableView, { FeaturesTableCallback } from './FeaturesTableView';
 import { Xref } from '../../../shared/types/apiModel';
+import NightingaleZoomTool from './NightingaleZoomTool';
```

Comment:
Is it possible to order imports in some kind of way? Not attached to how, but at least group things that make sense together.

I usually see:
 - external imports
 - components
 - hooks
 - utils
 - types
 - non-standard imports (.scss)
 
 But anything as long as when I open a file I can grasp what it depends on easily.
 See QueryBuilder.tsx or ResultsView.tsx for extreme examples.

Reply:
We should decide on a guideline for this together with @dlrice and document it here: https://github.com/ebi-uniprot/coding-guidelines

I'm happy with the one you're suggesting.

Reply:
If @dlrice doesn't see any objection I'll add it to the guidelines.
This should be a loose rule though, because it depends a lot on how crowded the imports are, and sometimes you do have to import different types of imports from the same files so the groups get mixed up (mainly thinking of TS types that might be within any kind of import).

Reply:
yep, loose rules (guidelines) are documented, while hard rules should be enforced by linting